source/about.rst

```diff
@@ -0,0 +1,26 @@
+About
+=====
+
+The Team
+--------
+
+Board of directors
+""""""""""""""""""
+- Timothy Berkelbach, Columbia and Flatiron Institute
+- Garnet Chan, Caltech
+- Sandeep Sharma, UC Boulder
+- Alexander Sokolov, Ohio State
+- Qiming Sun, Tencent Quantum Laboratory
+  
+Maintainers
+"""""""""""
+- Zhihao Cui, Caltech
+- Yang Gao, Caltech
+- Iliya Sabzevari, UC Boulder
+- James Smith, Flatiron Institute
+- Xiao Wang, Flatiron Institute
+
```

Comment:
Authors of docs?

Reply:
I think it's a great idea to credit everyone who's helped out with the docs, but if we do that I would feel bad not making another section for general contributors not already mentioned, and doing both of those is beyond the scope of this PR in my opinion.

Reply:
My email reply from below was referring to this: `I agree, I think something like that can be done later`